Global/Xcode.gitignore

```diff
@@ -15,9 +15,9 @@ DerivedData
 !default.mode2v3
 *.perspectivev3
 !default.perspectivev3
-xcuserdata
+xcuserdata/
 
 ## Other
-*.xccheckout
 *.moved-aside
-*.xcuserstate
+*.xccheckout
+*.xcscmblueprint
```

Comment:
This one seems to be new, but I'm not going to hold this up as it seems very specific.